lib/rubyhub/operations/pull_request/create.rb

```diff
@@ -17,6 +23,43 @@ def call(options)
 
           def data
             @data ||= Rubyhub::Configuration.instance.options.dig(:template, template.to_sym)
+            add_description_to_data
+            @data
```

Comment:
This is not good approach.
Check [this](https://github.com/rubygarage/rubyhub/blob/master/lib/rubyhub/pull_request.rb) file's method `message`. You should modify it to change current logic of adding descripion